python/ray/data/datasource/image_folder_datasource.py

```diff
@@ -133,6 +122,25 @@ def _read_file(self, f: "pyarrow.NativeFile", path: str, **reader_args):
             }
         )
 
+    def _check_import(self, *, module: str, package: str) -> None:
```

Comment:
nice, shall we put this inside [data/_internal/util.py](https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/util.py), so we can start use same method everywhere?

Reply:
Fixed